cmd/mavenExecute.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"bytes"
+	"io"
+	"strings"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+)
+
+const mavenExecutable = "mvn"
+
+func mavenExecute(config mavenExecuteOptions, telemetryData *telemetry.CustomData) string {
+	c := command.Command{}
+	output, err := runMavenExecute(&config, &c)
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+
+	return output
+}
+
+func runMavenExecute(config *mavenExecuteOptions, command execRunner) (string, error) {
```

Comment:
Just a suggestion: You could refactor this into a method which only builds the parameters array from the config, and then another which runs the maven executable. I got this idea because I was wondering why you create the command in mavenExecute() and pass it to runMavenExecute().

Reply:
sounds good